Models/MetAPI/GraphQLTypes/ForecastType.cs

```diff
@@ -12,6 +15,43 @@ public ForecastType()
             Field(forecast => forecast.ForecastProperties, false, typeof(PropertiesType))
                 .Description("Data and meta-data returned");
             Field(forecast => forecast.Type);
+
+            // What the actual metadata should look like, is hopefully compatible with most databases 
+            Field(forecast => forecast.StoredMetaData, false, typeof(StoredMetaDataType));
+
+            // Very bad and temporarily workaround connecting to a Atlas Cluster containing some metadata documents 
+            // connect using MongoClient
+            var dbClient =
```

Comment:
Could we perhaps do some dependency injection magic here, so we don't need one DBClient for each query?

Reply:
Yes we can! I believe that is the _way to go_.
We can retrieve the service in the resolvers and pass them down from there, IIRC.